Cargo.toml

```diff
@@ -36,6 +36,7 @@ dav1d-sys = { version = "0.2", optional = true }
 aom-sys = { version = "0.1.2", optional = true }
 scan_fmt = { version = "0.2", optional = true }
 ivf = { path = "ivf/", optional = true }
+ffmpeg-sys = { git = "https://github.com/meh/rust-ffmpeg-sys.git", branch = "master", optional = true }
```

Comment:
Is there no real crate?

Reply:
No, the crate only works with ffmpeg 3.x, we need  github repo to work with 4.x

Reply:
This could be problematic. Could you please split the set with the first 2 commit in another PR?

I'd land it immediately and then we can figure out how to get the libavformat muxing sorted out without git crates.

Reply:
sure, wait 2 mins

Reply:
done https://github.com/xiph/rav1e/pull/1304

Reply:
@lu-zero , code rebased.
How to handle this problem?
thanks

Reply:
I hadn't time to check which are the crates available. 

Wrapping libavformat isn't this difficult, so worst case we could make the (N+1)th one with just that purpose and put it on crates.io.

Reply:
Hi @lu-zero , thanks for suggestion. Any document/guidance for this?

thanks

Reply:
[here](https://rust-lang.github.io/rust-bindgen/tutorial-0.html) the tutorial, [here](https://github.com/rust-av/aom-rs/tree/master/aom-sys) a working example if you want to try yourself. Ping me on irc and I'll try to help.

Reply:
@lu-zero .
Thanks for offering the help. Seems it's not so difficult. I will try it by myself. And ping you if I have a hard issue.